Kros.Utils/src/Kros.Utils.MsAccess/Data/BulkActions/MsAccess/MsAccessBulkInsert.cs

```diff
@@ -368,8 +368,7 @@ private string CreateInsertSql(string tableName, string sourceFilePath)
 
         private string SqlCodePage
         {
-            get
-            {
+            get {
```

Comment:
Many opening braces is on the same line as get keyword. It is on purpose?

Reply:
No. Something changed my settings in Visual Studio.